### PR TITLE
Made action attributes changes and created tests for it

### PR DESCRIPTION
**Pull request by lucianopf**
- Added operation constants and must have attributes to create a new action
- Made attribution a bit more generic
- Check if all must have attributes are now given
- Check if the operation attribute is a valid based on operationConstants
- Made tests to new Action

### CODE REVIEW

File: src/action.js [hunk 1]
```diff
@@ -3,14 +3,21 @@
 let ParamsFilter = require('./param-filter')
 let Response = require('./response')
 let ModelLoader = require('./model-loader')
+let operationConstants = ['CREATE', 'UPDATE', 'DELETE', 'VIEW', 'LIST']
+let mustHaveAttributes = ['event', 'context', 'schema', 'operation', 'body']
```
Comment (andremetzen) [inline src/action.js]: I think it's better `requiredAttributes`. What do you think?

File: src/action.js [hunk 2]
```diff
@@ -3,14 +3,21 @@
...
-  this.body = config.body
-  this.schema = config.schema
+  for (var key in config) {
+    this[key] = config[key]
+    let oneMustHave = mustHaveAttributes.indexOf(key)
+    if (-~oneMustHave) {
+      mustHaveAttributes.splice(oneMustHave, 1)
+    }
+  }
+  if (mustHaveAttributes.length > 0 || ~-operationConstants.indexOf(this.operation)) {
```
Comment (andremetzen) [inline src/action.js]: What you're saying is that we won't have any others operations than those specified on `operationsConstants`, is that right? What if comes the need for a custom operation? How can we handle that?

Comment (lucianopf) [inline src/action.js]: Yes, that's a good question..
As I told you, I think we need some sort of configuration when the user is importing the module.

Something like:

```
let lamb = require('lamb-wrap')
lamb.customConfigs({
  operationsConstants: ['CUSTOM_OPERATION']
})
```

I've gotta figure it out first what will be the custom points of the module, but I guess this'll handle it.

After we decide this I can implement something as `operationsConstants.concat(customConfigs.operationsConstants)`

What's your opinion about that?

Comment (andremetzen) [inline src/action.js]: I think we should keep things simple at first, and improve when we face the need.

**Comment by andremetzen** [comment]
Can you write an example of how we would require the Action module and create a new Action object? I'm concerned about how we would use the operations constants...

**Pull request merged**